src/LoginFormContainer.test.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+import { fireEvent, render } from '@testing-library/react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import { changeLoginFields, setAccessToken } from './actions';
+
+import LoginFormContainer from './LoginFormContainer';
+
+jest.mock('react-redux');
+
+describe('LoginFormContainer', () => {
+  const dispatch = jest.fn();
+
+  beforeEach(() => {
+    dispatch.mockClear();
+
+    useDispatch.mockImplementation(() => dispatch);
+
+    useSelector.mockImplementation((selector) => selector({
+      loginFields: {
+        email: '',
+        password: '',
+      },
+      accessToken: given.accessToken,
+    }));
+  });
+
+  context('with accessToken', () => {
+    given('accessToken', () => ({
+      accessToken: 'qwer',
+    }));
+
+    it('render log out button', () => {
+      const { getByText } = render(<LoginFormContainer />);
+
+      fireEvent.click(getByText('Log out'));
+
+      expect(dispatch).toBeCalledWith(setAccessToken(null));
```

Comment:
```suggestion
      expect(dispatch).toBeCalledWith({ type: 'logout' });
```

어떤 액션 객체가 오는지 명시적으로 나타내면 표현력이 좋습니다.